Use built version os `sockjs-client`

filipesilva opened the pull request:
<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->

**What kind of change does this PR introduce?**
<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->
Bugfix

**Did you add or update the `examples/`?**
Yes.

**Summary**
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->
`webpack-dev-server` bundles `sockjs-client` from source. `sockjs-client` sources assume a node environment however, and will fail to run if `global` is not available.

By bundling `sockjs-client` from source, the node environment requirement from `sockjs-client` will also be required of the app being served and thus it is not possible to have `node.global = false` in the user app being bundled.

This fix uses the built `sockjs-client` instead of the sources.

**Does this PR introduce a breaking change?**
<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
No.

**Other information**
Fix https://github.com/webpack/webpack-dev-server/issues/1147

shellscape commented on examples/node-false/README.md:
```diff
@@ -0,0 +1,9 @@
+# CLI - node false
```
these are more for "how to" for users, but there's no harm in adding this. the tests that matter are in `/test`. no worries if you can't get to adding one there, we can do it after merging.

filipesilva commented:
@shellscape I removed the example, I don't think it's a real user example.

Regarding the real tests in `/test`, to test `node: false` doesn't throw a runtime error, a e2e test would be needed. It looks like there is no test where a browser is actually used. Instead I added `node: false` to the simple config to ensure that stuff in general still works with it, and that it's not needed for normal webpack-dev-server functioning. WDYT?

shellscape commented:
Yeah good point; we would actually need a browser for this. I'd leave the example in there as a means to test that change at the moment, even though it's not a traditional example. We can look into headless browser testing for the client in the future.

filipesilva commented:
Done, re-added it.

shellscape commented:
Thanks much :)

filipesilva commented:
Thank you for the quick review!
